src/tools/cephfs/cephfs-shell

```diff
@@ -583,7 +583,9 @@ exists.')
                         is_dir = True
                     else:
                         is_dir = False
-                if args.long and args.H:
+                if not args.all and path in ['.', '..']:
```

Comment:
Shouldn't the line 571 above actually filter these out?

Reply:
> Shouldn't the line 571 above actually filter these out?
In the present code, it does not.  I have sent a new patch to address it.